SUI/SUI/Products/ProductsView.swift

```diff
@@ -0,0 +1,225 @@
+    }
+
+    private var rowsView: some View {
+        ForEach(0 ..< productsViewModel.products.count, id: \.self) { item in
+            rowView(item: item)
+        }
+        .padding(.top, 10)
+    }
+
+    private var minusTextView: some View {
+        Text(Constants.minusString)
+            .foregroundColor(.black)
+            .font(.system(size: 30, weight: .bold, design: .default))
+    }
+
+    private var plusTextView: some View {
+        Text(Constants.plusString)
+            .foregroundColor(.black)
+            .font(.system(size: 25, weight: .bold, design: .default))
+    }
+
+    private var oldPriceTextView: some View {
+        Text(Constants.oldPriceString)
+            .strikethrough()
+            .foregroundColor(.lightGray)
+            .font(.system(size: 30, weight: .medium, design: .default))
+    }
+
+    // MARK: - Private methods
+
+    private func rowView(item: Int) -> some View {
+        ZStack {
+            backProductView
+            HStack {
+                Spacer(minLength: 40)
+                productImageView(product: productsViewModel.products[item])
+                Spacer(minLength: 10)
+                VStack {
+                    titleTextView(product: productsViewModel.products[item])
+                    HStack {
+                        minusButtonView(item: item)
+                        productCountTextView(product: productsViewModel.products[item])
+                        plusButtonView(item: item)
+                    }
+                }
+                Spacer(minLength: 20)
+                VStack {
+                    oldPriceTextView
+                    newPriceTextView(product: productsViewModel.products[item])
+                }
+                .padding(.trailing, 20)
+                Spacer()
+            }
+        }
+    }
+
+    private func minusButtonView(item: Int) -> some View {
+        Button {
+            productsViewModel.makeMinusCount(item: item)
+        } label: {
+            minusTextView
+        }
+    }
+
+    private func plusButtonView(item: Int) -> some View {
+        Button {
+            productsViewModel.makePlus(item: item)
+        } label: {
+            plusTextView
+        }
+    }
+
+    private func productImageView(product: Product) -> some View {
+        Image(product.imageName)
+            .resizable()
+            .foregroundColor(.orange)
+            .frame(width: 80, height: 70, alignment: .leading)
+    }
+
+    private func titleTextView(product: Product) -> some View {
+        Text(product.title)
+            .font(.system(size: 22, weight: .bold, design: .default))
+    }
+
+    private func newPriceTextView(product: Product) -> some View {
+        Text("\(Constants.dollarString)\(product.price)")
+            .foregroundColor(.darkPurple)
+            .font(.system(size: 40, weight: .medium, design: .default))
+            .frame(width: 100)
+    }
+
+    private func productCountTextView(product: Product) -> some View {
```

Comment:
```suggestion
    private func productCountTextView(product: Product) -> some View {
```
во всех методах make добавь